porfile.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfilo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+        <nav id="navbar">
+            <a class="logo" href="#">My Logo</a>
+            <ul>
+                <li><a href="">Portoflio</a></li>
+                <li><a href="">My Contact</a></li>
+                <li><a href="">About</a></li>
+            </ul>
+        <img src="./icons8-hamburger-32 (1).png" alt="" class="hamburger">
+</nav>
+            
+<div class="eli">
+    <img src="img/Rectangle 6.svg" alt="Rectangle 6" id="Rectangle6">
+    <img src="img/Ellipse 11.svg" alt="Ellipse 11" id="Ellipse11">
+    <img src="img/Rectangle 7.svg" alt="Rectangle 7" id="Rectangle7">
+    <img src="img/Ellipse 5.svg" alt="Ellipse 5" id="Ellipse5">
+    <img src="img/Rectangle 4.svg" alt="Rectangle 4" id="Rectangle4">
+    <img src="img/Ellipse 3.svg" alt="Ellipse 3" id="Ellipse3">
+
+</div>
+          
+    <section id="headline">
```

Comment:
- [ Required ] Please consider giving this section a `background-image` that is consistent with the `Figma` design, currently, the `background-image`, has some variations compared to the Figma design. I suggest using a `single` background image that has all the `shapes` present in the `Figma` design as a background image for this section. Please see the screenshot below for more details on this.

## Figma design
<img src="https://user-images.githubusercontent.com/16392046/219791833-3c821b6d-83c2-4836-9328-71b3b668bd79.png" width="375px" />

## Your design
<img src="https://user-images.githubusercontent.com/16392046/219792255-9779a519-bffe-4115-823c-4d38d0752a6d.png" />

## Background image to use
<img src="https://user-images.githubusercontent.com/16392046/219795053-8efd0698-a13a-4529-9a7d-f2bcc4bf9074.jpg" width="375px" />